Modules/Core/Common/include/itkHexahedronCell.hxx

```diff
@@ -581,9 +581,9 @@ HexahedronCell<TCellInterface>::EvaluateLocation(int &                     itkNo
   std::fill_n(x, Self::CellDimension, 0.0);
   for (unsigned int i = 0; i < Self::NumberOfPoints; i++)
   {
-    PointType pt = points->GetElement(m_PointIds[i]);
+    const PointType pt{ points->GetElement(m_PointIds[i]) };
```

Comment:
`PointType` is not explicitly defined in `itkHexahedronCell.h`.  Is it is coming from a superclass?  It might be more transparent to put
```c++
using PointType = TCellInterface::PointType;
```
in `itkHexahedronCell.h`.  (Or if `TCellInterface::` isn't where it is coming from, then use the correct location!)

Reply:
Not my code.  I don't want to learn the code just to silence compiler warnings.